src/NzbDrone.Core/Extras/Subtitles/SubtitleFile.cs

```diff
@@ -1,10 +1,28 @@
-﻿using NzbDrone.Core.Extras.Files;
+﻿using System.Collections.Generic;
+using System.IO;
+using NzbDrone.Core.Extras.Files;
 using NzbDrone.Core.Languages;
 
 namespace NzbDrone.Core.Extras.Subtitles
 {
     public class SubtitleFile : ExtraFile
     {
         public Language Language { get; set; }
+
+        public string AggregateString => Language + Extension + LanguageTags;
+
+        public string LanguageTags { get; set; }
+
+        public void SetLanguageTags(IEnumerable<string> newTags)
```

Comment:
Language tags can be set directly, no need for a function to set it.

Reply:
Removed in 24334c4.